components/effortless-chef-io/effortless-audit.md

```diff
@@ -0,0 +1,141 @@
+# Effortless Audit
+
+Effortless Audit is the pattern for managing you Chef InSpec profiles. IT uses [Chef Habitat](https://www.habitat.sh/docs/) and [Chef InSpec](https://www.inspec.io/) to build an artifact that contains your profile and it's dependencies alongside the scripts necessary to run them on you systems.
+
+> Note: To learn more about InSpec profiles checkout the documentation [here](https://www.inspec.io/docs/reference/profiles/).
+
+## Patterns
+
+### Wrapper Profile Pattern
+
+In InSpec a common pattern is to write a wrapper profile that depends on another profile. This pattern is used to pull profiles from a main profile source like the [Chef Automate Profile Store](https://automate.chef.io/docs/profiles/).
+
+> Note: You can find an example of this pattern [here](https://github.com/chef/effortless/tree/master/examples/effortless_audit).
+
+1. To use this pattern navigate to the profile you want to use
+   ```
+   cd my-profile
+   ```
+1. Make a habitat directory
+   ```
+   mkdir habitat
+   ```
+1. Make a plan file
+   
+   > Notes: For a profile targeting windows use a `plan.ps1` for Linux use a `plan.sh` if your profile targets both windows and linux you can have both a `plan.ps1` and a `plan.sh` in your habitat directory.
+   ```
+   touch plan.sh
+   ```
+1. Add some information about your profile to your plan
```

Comment:
Should we have a powershell example here too?

Reply:
Yep that makes sense I'll add that